tsconfig.json

```diff
@@ -14,8 +14,6 @@
 		],
 		"module": "umd",
 		"moduleResolution": "node",
-		"noImplicitAny": true,
```

Comment:
Has this been replaced with a different compiler option somewhere?

Reply:
Yes, `strict` covers `noImplicitAny`, `noImplicitThis`, `alwaysStrict`, `strictNullChecks`, and `strictFunctionTypes`.